source/tutorials/openmm.rst

```diff
@@ -17,4 +23,128 @@ folder of the GitHub repository. It contains:
   NanoVer simulation using a python client.
 * `openmm_neuraminidase`: A notebook that demonstrates how to set up an interactive
   OpenMM simulation from scratch with NanoVer and change the visualisations of
-  different atom selections for a protein-ligand system.
\ No newline at end of file
+  different atom selections for a protein-ligand system.
+
+XML Files
+==============================================
+
+The XML format for serializing OpenMM simulations consists of a root element ``<OpenMMSimulation>`` containing three main components:
+
+1. **Starting Structure**: Enclosed in either ``<pdbx>`` or ``<pdb>`` tags
+2. **OpenMM Serialized System**: Enclosed in the ``<System>`` tag
+3. **OpenMM Serialized Integrator**: Enclosed in the ``<Integrator>`` tag
+
+An optional fourth component, the OpenMM serialized state, may also be included.
+The format allows for efficient storage and exchange of OpenMM simulation data, including the starting structure, system configuration, and integrator settings.
```

Comment:
Can you link to some documentation about what an "OpenMM serialized state" is?

Reply:
I read about it in the openmm documentation [here](http://docs.openmm.org/7.7.0/userguide/theory/05_other_features.html?highlight=serialized%20state#xmlserializer) but nothing more exists about it. I wanted to keep the information there in case someone know what it is and needs it.

Reply:
I think that OpenMM just has a method of serializing the objects it uses (be is a system, integrator, state etc.), and the label "OpenMM serialized" just indicates that it's been created using the functionality of OpenMM. So rather than explaining what an OpenMM serialized state is, it may be better to put in a short sentence to say that the fields prefixed by "OpenMM serialized" are created using the OpenMM inbuilt functionality to do so.

Reply:
Solved in this commit [0043457](https://github.com/IRL2/nanover-docs/pull/115/commits/0043457c7c21028f743a19611dc6a729dd6a82c8)